.circleci/config.yml

```diff
@@ -106,9 +156,28 @@ jobs:
       - run:
           name: Scan credentials
           command: sh scan.sh
+  upload_to_s3_and_invalidate_cache:
+    executor: ci-aws
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - aws-cli/setup:
+          profile-name: default
+          configure-default-region: false
+      - aws-cli/role-arn-setup:
+          profile-name: ci-web-prod
+          source-profile: default
+          role-arn: arn:aws:iam::049322569699:role/dashboard-ci-web-prod
+      - aws-s3-sync:
+          from: 'dist/'
+          to: 's3://genai-aichatbotwidget-use1-prod/'
+          arguments: '--delete'
+      - aws-create-invalidation:
```

Comment:
Just in case, don't we need to run this only when there is an explicit user action, such as tagging?

Reply:
I turned it off for pipeline testing. I turned It on